apps/explorer/src/explorer/pages/Home/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
-import { useFeatureFlags } from '@cowprotocol/common-hooks'
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
+import { useFlags } from 'launchdarkly-react-client-sdk'
```

Comment:
I understand the reasoning from the PR description.

In that case, shouldn't this be removed from the app as well?

https://github.com/cowprotocol/cowswap/blob/develop/apps/explorer/tsconfig.json#L15

Alternatively, this is overkill, but we could also export it as an independent lib to share potential feature flag logic between both apps.

Reply:
<img width="1163" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/1a40e19b-d2be-4cc2-92c8-5d054ee45b56">

The main reason why I did this change is because Explorer build started failing.
As you can see it starts getting `@cowprotocol/wallet-provider` transitively.
I think we should refactor `@cowprotocol/common-hooks` and remove cowswap specific parts from it.

Reply:
Yes, that's what I meant by `export it as an independent lib`